source/simulator/simulator_signals.cc

```diff
@@ -28,7 +28,8 @@ namespace aspect
 {
   // provide locations for the static variables of this class
   template <int dim>
-  boost::signals2::signal<void (ParameterHandler &)>  SimulatorSignals<dim>::declare_additional_parameters;
+  boost::signals2::signal<void (const unsigned int,
```

Comment:
Maybe I do not understand the structure behind these signals, but if we have `<int dim>` as template argument, why do we need another unsigned int as argument to this function?